packages/@react-aria/test-utils/src/menu.ts

```diff
@@ -172,6 +216,28 @@ export class MenuTester {
     return null;
   };
 
+  keyboardNavigateToOption = async (opts: {option: HTMLElement}) => {
```

Comment:
I haven't gotten to the actual tests but should this be exposed to the user or kept as a utility in this tester? If we do want to keep this exposed, we should discuss what level of intermediate interactions should be created and what audience would find them useful. My initial imagining of the utils was that they'd be relatively opaque with each interaction being a relatively self contained "user flow" of sorts, but that vision was aimed more at test writers who are testing against an application rather than a component library. We could definitely aim to provide utilities for both tho

Reply:
I don't have a strong preference on if this is public or private, happy to make private for now and we can always expose it later

Reply:
yup, perhaps a bit too ambitious for now, private seems good to me for now